views/group/show.pug

```diff
@@ -119,6 +131,16 @@ block content
             a(href=`/users/${user.id}` class='lg:whitespace-no-wrap hover:text-blue-500')= user.name
             if user.id === group.ownerId
               span(class='px-3 py-1 text-sm text-white bg-purple-500 rounded-full') Szervező
+            else if isOwner || isAdmin
+              button(type='button' class='px-3 py-1 text-sm text-white btn btn-primary animate-hover cursor-pointer' onClick=`toggleModal(
```

Comment:
I think it would look better if these buttons were placed to the right side (like using justify-between), so they won't be just floating around based on the username's length. Also on mobile the names tend to stick to each other, so a bigger vertical spacing / smaller font size could help.